### ISSUE TITLE
->parsefile() generates an error.

### ISSUE THREAD

**Issue by shlomif** [petdance/html-lint#58]
Hi!

As this test case demonstrates using `->parsefile()` instead of `->parse()` generates an `The parse() method has not been called on this file.'` error in recent HTML::Lints:

```perl
use strict;
use warnings;
use autodie;

use Test::More tests => 1;

use HTML::Lint;

my $OUTPUT_FN = "test.xhtml";

open my $o, '>', $OUTPUT_FN;
print {$o} <<'EOF';
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xml:lang="en-US" xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>Foo</title>
<meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
</head>
<body>
<h1>Hello</h1>
<p>File</p>
</body>
</html>
EOF
close($o);
my $lint = HTML::Lint->new;

$lint->parse_file($OUTPUT_FN);

# TEST
is_deeply ([map { $_->as_string() } $lint->errors()], [],
    "HTML is valid for output file."
);

```

**Comment by petdance** [comment]
Fixed.  Released in 2.27_01, the beta for 2.28.

**Comment by shlomif** [comment]
On Fri, 22 Dec 2017 13:54:49 -0800
Andy Lester <notifications@github.com> wrote:

> Fixed.  Released in 2.27_01, the beta for 2.28.
> 

thanks!

-- 
-----------------------------------------------------------------
Shlomi Fish       http://www.shlomifish.org/
Perl Elems to Avoid - http://perl-begin.org/tutorials/bad-elements/

There are at least 137 Larry Walls in the U.S., but only one that matters.
    — http://www.shlomifish.org/humour/bits/facts/Larry-Wall/

Please reply to list if it's a mailing list post - http://shlom.in/reply .
